pkgs/tools/inputmethods/ibus-engines/ibus-rime/default.nix

```diff
@@ -0,0 +1,80 @@
+{ stdenv
+, fetchFromGitHub
+, wrapGAppsHook
+, gdk-pixbuf
+, pkgconfig
+, cmake
+, librime
+, ibus
+, libnotify
+, brise
+}:
+
+stdenv.mkDerivation rec {
+  pname = "ibus-rime";
+  version = "1.4.0";
+
+  src = fetchFromGitHub {
+    owner = "rime";
+    repo = "ibus-rime";
+    rev = version;
+    sha256 = "0zbajz7i18vrqwdyclzywvsjg6qzaih64jhi3pkxp7mbw8jc5vhy";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkgconfig
```

Comment:
In new expressions, `pkg-config` should be used instead of the `pkgconfig` alias.